packages/monorepo-builder/src/DevMasterAliasUpdater.php

```diff
@@ -25,16 +27,25 @@ final class DevMasterAliasUpdater
     /**
      * @var string
      */
-    private const DEV_MASTER = 'dev-master';
+    private const COMPOSER_BRANCH_PREFIX = 'dev-';
```

Comment:
I went with defining constants here to try and stay inline with what was done in this file.
Although 'dev-' is only used in one place so it could also be inlined.